be/src/vec/core/block.h

```diff
@@ -348,6 +348,13 @@ class MutableBlock {
         _data_types.clear();
     }
 
+    void append_from_block(const Block* block, const int col_len, const int start_col_idx, const size_t row_number) {
```

Comment:
the function do not call any where? should not add here

Reply:
i agree，i will delete it.